Makefile

```diff
@@ -91,23 +93,25 @@ $(SI_PROTO).tmp: $(SI_SPEC)
 	(diff $@ $(SI_PROTO) > /dev/null 2>&1 || mv -f $@ $(SI_PROTO)) && \
 		rm -f $@
 
-$(CONSTANTS_GO).tmp: $(SI_SPEC)
+$(CONSTANTS_TMP): $(SI_SPEC)
 	test -d $(COMMON_LIB) || mkdir -p $(COMMON_LIB)
 	@echo "$$GENERATED_HEADER" > $@
 	@echo "$$PACKAGE_DEF" >> $@
-	cat $< | sed -n -e '/``constants$$/,/^```$$/ p' | sed '/^```/d' >> $@
+	cat $< | sed -n -e '/```constants$$/,/^```$$/ p' | sed '/^```/ s/.*//g' >> $@
+	go fmt $@
 	(diff $@ $(CONSTANTS_GO) > /dev/null 2>&1 || mv -f $@ $(CONSTANTS_GO)) && \
 		rm -f $@
 
-$(INTERFACE_GO).tmp: $(SI_SPEC)
+$(INTERFACE_TMP): $(SI_SPEC)
 	test -d $(API_LIB) || mkdir -p $(API_LIB)
 	@echo "$$GENERATED_HEADER" > $@
-	cat $< | sed -n -e '/``golang$$/,/^```$$/ p' | sed '/^```/d' >> $@
+	cat $< | sed -n -e '/``golang$$/,/^```$$/ p' | sed '/^```/ s/.*//g' >> $@
```

Comment:
Why we got ``golang$$, not ```golang$$$ which with three grave accent?
The constants with three grave accent above.

Reply:
I thought I had fixed that one too, looks like I missed it.